rotkehlchen/db/upgrades/v32_v33.py

```diff
@@ -71,6 +75,385 @@ def _create_new_tables(cursor: 'DBCursor') -> None:
+        ), C6 AS (
+            SELECT
+                lasttoken1 AS base1,
+                firsttoken1 AS quote1,
+                last1out AS amount1,
+                CAST(first1in AS REAL) / CAST(last1out AS REAL) AS rate1,
+                lasttoken0 AS base2,
+                firsttoken0 AS quote2,
+                last0out AS amount2,
+                CAST(first0in AS REAL) / CAST(last0out AS REAL) AS rate2,
+                txhash, logindex, time, location
+            FROM amounts_query
+            WHERE first1in > 0 AND first0in > 0 AND last1out > 0 AND last0out > 0
+        ), SWAPS AS (
+        SELECT base1 AS base_asset, quote1 AS quote_asset, amount1 AS amount, rate1 AS rate, txhash, logindex, time, location FROM C1
+        UNION ALL /* using union all as there can be no duplicates so no need to handle them */
+        SELECT base1 AS base_asset, quote1 AS quote_asset, amount1 AS amount, rate1 AS rate, txhash, logindex, time, location FROM C2
+        UNION ALL /* using union all as there can be no duplicates so no need to handle them */
+        SELECT base1 AS base_asset, quote1 AS quote_asset, amount1 AS amount, rate1 AS rate, txhash, logindex, time, location FROM C3
+        UNION ALL /* using union all as there can be no duplicates so no need to handle them */
+        SELECT base1 AS base_asset, quote1 AS quote_asset, amount1 AS amount, rate1 AS rate, txhash, logindex, time, location FROM C4
+        UNION ALL /* using union all as there can be no duplicates so no need to handle them */
+        SELECT base1 AS base_asset, quote1 AS quote_asset, amount1 AS amount, rate1 AS rate, txhash, logindex, time, location FROM C5
+        UNION ALL /* using union all as there can be no duplicates so no need to handle them */
+        SELECT base2 AS base_asset, quote2 AS quote_asset, amount2 AS amount, rate2 AS rate, txhash, logindex, time, location FROM C5
+        UNION ALL /* using union all as there can be no duplicates so no need to handle them */
+        SELECT base1 AS base_asset, quote1 AS quote_asset, amount1 AS amount, rate1 AS rate, txhash, logindex, time, location FROM C6
+        UNION ALL /* using union all as there can be no duplicates so no need to handle them */
+        SELECT base2 AS base_asset, quote2 AS quote_asset, amount2 AS amount, rate2 AS rate, txhash, logindex, time, location FROM C6
+    )
+    SELECT
+        txhash + logindex AS id,
+        time,
+        location,
+        base_asset,
+        quote_asset,
+        'A' AS type, /* always a BUY */
+        amount,
+        rate,
+        NULL AS fee, /* no fee */
+        NULL AS fee_currency, /* no fee */
+        txhash AS link,
+        NULL AS notes /* no notes */
+    FROM SWAPS
+    UNION ALL /* using union all as there can be no duplicates so no need to handle them */
+    SELECT * from trades;
+    """)  # noqa: 501
+
+
+def _change_event_identifier_to_bytes(cursor_fetch: 'DBCursor', cursor_update: 'DBCursor') -> None:
```

Comment:
Can you add a docstring here? On what this part of the upgrade does. Same for `_force_bytes_for_tx_hashes`. In fact since they are both part of the same part of the upgrade why not combine them in one function?  I mean splitting in 2 is fine but when called in `upgrade_v32_to_v33` its' better to have it as one function called since it's one thing getting upgraded.